Week2/homework/maartjes-work.js

```diff
@@ -47,15 +47,21 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const maartjesTasks = tasks.map(elem => ({ ...elem, duration: elem.duration / 60 }));
```

Comment:
`maartjesTasks` is already declared on line 45, be careful with variable names, this is also giving you an error in the linter (run `npm run lint` in your terminal to see).

Reply:
The `map()` function already returns the elem so you could only change the duration.

`tasks.map(elem => elem.duration / 60);`